app/models/school_class.rb

```diff
@@ -2,12 +2,15 @@
 
 class SchoolClass < ApplicationRecord
   belongs_to :school
-  has_many :members, class_name: :ClassMember, inverse_of: :school_class, dependent: :destroy
+  has_many :students, class_name: :ClassStudent, inverse_of: :school_class, dependent: :destroy
+  has_many :class_teachers, class_name: :ClassTeacher, inverse_of: :school_class, dependent: :destroy
```

Comment:
why use `:students` and then `:class_teachers` instead of `:teachers` or vice versa? going with one or the other would be nice for consistency

Reply:
Ah yes, that would simplify things, thanks 👍